### PR TITLE
Update PO spec to align with RFC

### PR DESCRIPTION
**Pull request by CLEckhardt**
Resolve divergence between purchase order specification page in
v0.3 and RFC.

Signed-off-by: Chris Eckhardt <eckhardt@bitwise.io>

### CODE REVIEW

**Comment by CLEckhardt** [comment]
Here is the RFC: https://github.com/Cargill/grid-rfcs/blob/ryanlassigbanks-purchase-order-rfc/text/0025-purchase-order.md

File: docs/0.3/purchase_order_family_specification.md [hunk 1]
```diff
@@ -37,42 +35,57 @@ Order.
...
+      // Alternate identifiers for this purchase order
+      repeated PurchaseOrderAlternateId alternate_ids = 7;
       // Time the purchase order was created.
-      uint64 created_at = 5;
+      uint64 created_at = 8;
       // True if the purchase order was closed, false otherwise
-      bool is_closed = 6;
+      bool is_closed = 9;
+      // The name of the workflow of this purchase order
+      string workflow_type = 10;
```
Comment (shannynalayna) [inline docs/0.3/purchase_order_family_specification.md:71]: Please update this spec to align with the most current version of the RFC

Comment (CLEckhardt) [reply docs/0.3/purchase_order_family_specification.md]: Done

File: docs/0.3/purchase_order_family_specification.md [hunk 2]
```diff
@@ -170,60 +187,67 @@ address of:
...
 }
 
 message UpdateVersionPayload {
   string version_id = 1;
-  string workflow_status = 2;
-  bool is_draft = 3;
-  string current_revision_id = 4;
+  string po_uid = 2;
+  string workflow_state = 3;
+  bool is_draft = 4;
```
Comment (shannynalayna) [inline docs/0.3/purchase_order_family_specification.md:250]: Isn't the 'current_revision_id' still a field in this payload ?

Comment (shannynalayna) [reply docs/0.3/purchase_order_family_specification.md:250]: Nevermind, was confusing this one with the po version state object -_-

**Review by shannynalayna — APPROVED** [review]

**Review by dnewh — APPROVED** [review]

**Pull request merged**